enrolments/validators.py

```diff
@@ -48,6 +48,34 @@ def validate_attendance(classObj):
             )
 
 
+def validate_fields(session_obj):
```

Comment:
is the input for this function a session or a field object?

Reply:
it's a field object, will update the naming accordingly!